### DISCUSSION TITLE
Replay of commands over any sheet

### DISCUSSION THREAD

**Discussion by hacker-DOM** [saulpw/visidata#1950] [General]
Hey guys, I was wondering if it is possible to execute commands from the Cli (kind of like you can do in Vim with the `+'bla'` or `-c bla` Cli options. Essentially I have something like

```zsh
function dom_gho() {
    local owner="$1"
    shift
    gh repo list "$owner" --limit 150 --json nameWithOwner,languages,stargazerCount,createdAt,pushedAt,updatedAt,description,homepageUrl,name,url,visibility,isArchived "$@" | vd -f json -
}
```

I am writing a wrapper over `gh repo list <organization>` but would like to sort by `stargazerCount` automatically. Unfortunately it seems to me that this isn't possible even with replays since they specify which sheet to open. Thanks in advance for ideas.

**Comment by saulpw** [comment]
Hi @hacker-DOM, it is possible with replays, you just need to remove the open-file command, and the replay will operate on the current (top) sheet.

**Reply by hacker-DOM** [reply]
That worked, thanks! Is there a way to also set the width of the `nameWithOwner` column to 60 units?

**Reply by hacker-DOM** [reply]
Actually, weirdly enough, it stopped working. Here is my data:  https://gist.github.com/hacker-DOM/8d851bc19e83ade3308c6dbcec7dd9b0  and I am running it with vd --play dom_gho_sort_by_stargazers_file.vd dom_gho.csv. I have double checked and the names are equal ('stargazerCount' == 'stargazerCount')

I get `no "stargazerCount" column | replay cancelled`.

**Reply by saulpw** [reply]
What happens if you add a wait between statements like `-w 0.2`?

**Reply by hacker-DOM** [reply]
Hey @saulpw , it just doesn't work. I've created an page to track this issue: https://github.com/saulpw/visidata/issues/1951.
